libr/anal/d/cc-avr-8

```diff
@@ -0,0 +1,26 @@
+default.cc=avr
+
+avr=cc
+cc.avr.name=avr
+cc.avr.argn=r26
```

Comment:
Gotcha, can I then remove `optlink`, `stdcall`, `fastcall` and just leave `avr=cc` section with the changes you propose? Would that work?